Undeclared identifier 'tbb'

Hi @Rombur the last version of deal.ii and it does not compile. I think that it is related to  #7808 

The compilation report with the errors can be found here:
http://cdash.kyomu.43-1.org/buildSummary.php?buildid=8495

A few of errors:
```make
In file included from /opt/src/dealii/include/deal.II/base/aligned_vector.h:24:                                                                                                                                      
/opt/src/dealii/include/deal.II/base/parallel.h:160:7: error: use of undeclared identifier 'tbb'           
      tbb::parallel_for(tbb::blocked_range<Iterator>(x_begin, x_end, grainsize),                                                                                                   
      ^                                                                                                                                                                                 
/opt/src/dealii/include/deal.II/base/parallel.h:160:25: error: use of undeclared identifier 'tbb'                                            
      tbb::parallel_for(tbb::blocked_range<Iterator>(x_begin, x_end, grainsize),                              
                        ^                                                                                         
/opt/src/dealii/include/deal.II/base/parallel.h:160:44: error: 'Iterator' does not refer to a value         
      tbb::parallel_for(tbb::blocked_range<Iterator>(x_begin, x_end, grainsize),                                                             
                                           ^                                                        
/opt/src/dealii/include/deal.II/base/parallel.h:153:24: note: declared here                                                           
    template <typename Iterator, typename Functor>                                                                 
                       ^                                                                                                                                                     
/opt/src/dealii/include/deal.II/base/parallel.h:162:25: error: use of undeclared identifier 'tbb'                                                                     
                        tbb::auto_partitioner());                                                                                                                                                
                        ^ 
```

--- a/include/deal.II/base/parallel.h
+++ b/include/deal.II/base/parallel.h
@@ -147,6 +147,7 @@ namespace parallel
 
 
 
+#ifdef DEAL_II_WITH_THREADS
     /**
      * Encapsulate tbb::parallel_for.
      */
@@ -179,6 +180,7 @@ namespace parallel
                         functor,
                         *partitioner);
     }
+#endif
   } // namespace internal
 
   /**

